src/script/media/MediaStreamHandler.ts

```diff
@@ -72,6 +72,28 @@ export class MediaStreamHandler {
     }
   }
 
+  /**
+   * The method creates a media stream to enforce access rights to the camera and the microphone. If access is not possible, it starts a user pop-up
+   * @param video When `video=true` then the camera is also addressed. In many cases this is not necessary, because one
+   * track is enough to enforce the general permissions.
+   * @returns Promise with active MediaStream
+   */
+  requestMediaSreamAccess(video: boolean): Promise<MediaStream> {
+    const audio = true;
+    const screen = false;
+    return window.navigator.mediaDevices
+      .getUserMedia({audio, video})
+      .then((mediaStream: MediaStream) => {
+        return mediaStream;
+      })
+      .catch((error: Error) => {
+        if (isMediaStreamReadDeviceError(error.name)) {
+          this.schedulePermissionHint(audio, video, screen);
+        }
+        throw error;
```

Comment:
Try to do early return instead of checking inside. :)

```suggestion
        if (!isMediaStreamReadDeviceError(error.name)) {
          throw error;
        }
        this.schedulePermissionHint(audio, video, screen);
```